### PR TITLE
chore: Add paparazzi tests fro textfield

### LINKED ISSUE #478: Add paparazzi test for every TextField state possible

### PR DESCRIPTION
**Pull request by soulcramer**
## 📋 Changes description

Add generated test thanks to Parameterized to create a screenshot for most of textfield states

## 📸 Screenshots

Visible in the files (be aware that it might crash your browser as there's a lot of them)

## 🗒️ Other info

Close #478

### CODE REVIEW

**Review by shkschneider — DISMISSED** [review]
LGTM

File: spark-screenshot-testing/src/test/snapshots/images/com.adevinta.spark.textfields_TextFieldScreenshot_TextFieldTest[107]__state[basic]_value[11]_required[true]_leadingicon[false]_trailingicon[true]_enabled[false]_readonly[true]_helper[121].png
Comment (MarinaRomanova) [inline spark-screenshot-testing/src/test/snapshots/images/com.adevinta.spark.textfields_TextFieldScreenshot_TextFieldTest[107]__state[basic]_value[11]_required[true]_leadingicon[false]_trailingicon[true]_enabled[false]_readonly[true]_helper[121].png:1]: Helper does not change it's color in disabled state?

File: spark-screenshot-testing/src/test/snapshots/images/com.adevinta.spark.textfields_TextFieldScreenshot_TextFieldTest[103]__state[basic]_value[11]_required[true]_leadingicon[true]_trailingicon[true]_enabled[true]_readonly[true]_helper[121].png
Comment (MarinaRomanova) [inline spark-screenshot-testing/src/test/snapshots/images/com.adevinta.spark.textfields_TextFieldScreenshot_TextFieldTest[103]__state[basic]_value[11]_required[true]_leadingicon[true]_trailingicon[true]_enabled[true]_readonly[true]_helper[121].png:1]: Looks like there are some duplicates. I am not sure what is the difference between this screenshot `TextFieldTest[101]__state[basic]_value[11]_required[true]_leadingicon[true]_trailingicon[true]_enabled[true]_readonly[false]_helper[121].png`(just above)

File: spark-screenshot-testing/src/test/snapshots/images/com.adevinta.spark.textfields_TextFieldScreenshot_TextFieldTest[107]__state[basic]_value[11]_required[true]_leadingicon[false]_trailingicon[true]_enabled[false]_readonly[true]_helper[121].png
Comment (soulcramer) [reply spark-screenshot-testing/src/test/snapshots/images/com.adevinta.spark.textfields_TextFieldScreenshot_TextFieldTest[107]__state[basic]_value[11]_required[true]_leadingicon[false]_trailingicon[true]_enabled[false]_readonly[true]_helper[121].png:1]: Yep, it stays visible in figma

**Pull request merged**